waspc/src/Wasp/Generator/ServerGenerator.hs

```diff
@@ -126,6 +127,17 @@ npmDepsForWasp spec =
           ]
     }
 
+depsRequiredBySessions :: AppSpec -> [AS.Dependency.Dependency]
```

Comment:
maybe `depsRequiredByAuth`?

Reply:
I kinda liked the idea of noting they are for sessions, even though they are automatically enabled with auth (for now). So while auth depends on session, session does not depend on on auth (in general), and in the future perhaps we enable session support even when auth is not on?

Unsure what is meant by elevate the check. Can you please elaborate a bit? Thanks

Reply:
Ok, makes sense! What would be the scenario though where session support is on but auth is not? For tracking some info about users even if they are not authenticated?

By elevating I meant moving it to the caller of this function, the function that calls it.

Reply:
Flash messages are typically stored in some sort of session. There may be other things that are not strictly auth related as well. Mainly thinking just from a dependency point of view though.

Ah ok, I see what you mean. I may just keep the auth check local to the fn and add a note about this possible use case and breaking that dependency with auth in the future.